### DISCUSSION TITLE
Now I will give zsh-snap a go

### DISCUSSION THREAD

**Discussion by AnimiVulpis** [marlonrichert/zsh-snap#126] [Show and tell]
I don't know if @marlonrichert  will remember me.
We interacted only through an issue created in the original `zinit` repository where https://github.com/marlonrichert/zsh-autocomplete would not be able to be loaded correctly.

I still have the issue history in my mails, but the original user and repository are gone.

More details [here](https://github.com/zdharma-continuum/I_WANT_TO_HELP) (Also look at the reddit thread linked there if still accessible)

TLDR: `zdharma` is gone and all his tools also disappeared from GitHub.

**Sounds like a good time to try out** `zsh-snap` :+1: 

To quote @marlonrichert :wink: 

> Or you could switch to plugin manager that doesn't have this problem... *cough*⚡️Znap*cough* 😉

**Comment by vladdoster** [comment]
@AnimiVulpis  see [zdharm-contiuum](https://github.com/zdharma-continuum)
